packages/eslint-plugin/lib/rules/no-restricted-attr-values.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * @typedef {import("../types").Rule} Rule
+ * @typedef {{attrPatterns: string[], attrValues: string[], message?: string}[]} Options
+ */
+
+const { RULE_CATEGORY } = require("../constants");
+
+const MESSAGE_IDS = {
+  RESTRICTED: "restricted",
+};
+
+/**
+ * @type {Rule}
+ */
+module.exports = {
+  meta: {
+    type: "code",
+
+    docs: {
+      description: "Disallow specified attributes",
+      category: RULE_CATEGORY.BEST_PRACTICE,
+      recommended: false,
+    },
+
+    fixable: null,
+    schema: {
+      type: "array",
+
+      items: {
+        type: "object",
+        required: ["attrPatterns", "attrValues"],
+        properties: {
+          attrPatterns: {
+            type: "array",
+            items: {
+              type: "string",
+            },
+          },
+          attrValues: {
+            type: "array",
+            items: {
+              type: "string",
+            },
+          },
+          message: {
+            type: "string",
+          },
+        },
+      },
+    },
+    messages: {
+      [MESSAGE_IDS.RESTRICTED]:
+        "'{{attrValues}}' is restricted from being used.",
+    },
+  },
+
+  create(context) {
+    /**
+     * @type {Options}
+     */
+    const options = context.options;
+    const checkers = options.map((option) => new PatternChecker(option));
+
+    return {
+      [["Tag", "StyleTag", "ScriptTag"].join(",")](node) {
+        node.attributes.forEach((attr) => {
+          if (!attr.key || !attr.value?.value || !attr.key?.value) {
```

Comment:
```suggestion
          if (!attr.key || typeof attr.value?.value !== "string" || !attr.key?.value) {
```
I think we should check when the attribute value is empty string. `<img alt="" />`